projects/program-website/src/app/app-routing.module.ts

```diff
@@ -0,0 +1,20 @@
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+import {ProjectsComponent} from '@/projects-component/projects.component';
+import {AboutComponent} from '@/about-component/about.component';
+import {ContactUsComponent} from '@/contact-us-component/contact-us.component';
+import {ParticipantsComponent} from '@/participants-component/participants.component';
```

Comment:
BTW, could you use constant format style in your code?
If first 2 imports has space between `{}`, then why other don't?
You can fix it automatically by code formatter in your code editor.
VS code: Shift+Ctrl+I
Sublime: Ctrl+T+F